### ISSUE TITLE
NaN during training

### ISSUE THREAD

**Issue by rorycawley** [gm-spacagna/deep-ttf#3]
I started to see NaN during training at about 20 epochs, is the nanterminator supposed to prevent that? The output was all NaN.

**Comment by gm-spacagna** [comment]
The nan terminator helps you stopping as soon as you hit the nan loss but won't help preventing it.

Which backend are you using? Sometime you need to change the epsilon parameter of tensorflow like it is described here:
https://github.com/gm-spacagna/deep-ttf#debugging-nan-weights
Have you tried?
